server/domain/academic-record/models/AcademicRecord.ts

```
@@ -0,0 +1,182 @@
+import { AcademicSummary } from './AcademicSummary';
+import { CourseEnrollment } from './CourseEnrollment';
+import { CourseEnrollments } from './CourseEnrollments';
+import { GradeType } from './Grade';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)